src/Validator/Improvers/hdtNIFSkinMeshValidator.cpp

```
@@ -0,0 +1,525 @@
+					std::memcpy(&v, block.data() + off, 4);
+					if (v >= 0 && v < numBlocks)
+						outbound[static_cast<size_t>(i)].push_back(v);
+				}
+			}
+
+			std::vector<SkinMeshCandidate> out;
+			for (int32_t i = 0; i < numBlocks; ++i) {
+				auto tOpt = blockTypeOf(parsed, i);
+				if (!tOpt)
+					continue;
+				if (*tOpt != nif::kTypeBSTriShape && *tOpt != nif::kTypeBSDynamicTriShape)
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)